curator.py

```diff
@@ -0,0 +1,105 @@
+# coding: utf8
+
+import os
+import sys
+import json
+from elasticsearch import Elasticsearch
+from datetime import date
+from datetime import timedelta
+
+# read environment variables
+elasticsearch_host = os.getenv("ELASTICSEARCH_HOST", "elasticsearch:9200")
+retention_days = int(os.getenv("RETENTION_DAYS", "14"))
+index_name_prefix = os.getenv("INDEX_NAME_PREFIX", "fluentd-")
+index_name_timeformat = os.getenv("INDEX_NAME_TIMEFORMAT", "%Y.%m.%d")
+
+
+def log(level="info", message="", extra=None):
+    """
+    Prints a JSON-formatted log message
+    """
+    msg = {
+        "level": level,
+        "message": message
+    }
+    if extra is not None:
+        msg["extra"] = extra
+    print(json.dumps(msg))
```

Comment:
python 2, but python 3 printing?

Reply:
`print()` is valid in python 2 and good practice, as it's upwards-compatible.